src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -664,6 +731,10 @@ def load_arguments(self, _):
             c.argument('yes', options_list=[
                        '--yes', '-y'], help='Do not prompt for confirmation.', action='store_true')
 
+    with self.argument_context('aks trustedaccess role') as c:
+        c.argument('list', required=True,
+                   help='List the trusted access roles of the subscription')
+
```

Comment:
I don't think `list` is a parameter. This file is declaring parameters. Maybe something like
```suggestion
    with self.argument_context('aks trustedaccess') as c:
        pass
```